trollmoves/client.py

```diff
@@ -407,6 +407,10 @@ def reload_config(filename, chains, callback=request_push, pub_instance=None):
             if val.get("heartbeat", False):
                 topics.append(SERVER_HEARTBEAT_TOPIC)
             for provider in chains[key]["providers"]:
+                if '/' in provider.split(':')[-1]:
```

Comment:
shouldn't this be:
```suggestion
                if '/' in provider.split(':')[1]:
```

Reply:
Here the `provider` has `tcp://` prefixed, so the last part of the provider string needs to be checked for the topic.

Reply:
Then you need to make sure that there is a scheme in the provider url
Maybe using urlsparse could help making sure we look at the right part ?